recipes/mmtf-pyspark/bld.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
You can drop the `bld.bat` and `build.sh` and use `pip` in the `meta.yaml` instead:

https://conda-forge.org/docs/meta.html#use-pip